tests/model_serving/model_server/authentication/test_kserve_token_authentication_raw.py

```diff
@@ -45,3 +45,32 @@ def test_disabled_raw_model_authentication(self, patched_remove_raw_authenticati
             model_name=ModelFormat.CAIKIT,
             use_default_query=True,
         )
+
+    @pytest.mark.dependency(depends=["test_disabled_raw_model_authentication"])
+    def test_re_enabled_raw_model_authentication(self, http_s3_caikit_raw_inference_service, http_raw_inference_token):
+        """Verify model query after authentication is re-enabled"""
+        verify_inference_response(
+            inference_service=http_s3_caikit_raw_inference_service,
+            inference_config=CAIKIT_TGIS_INFERENCE_CONFIG,
+            inference_type=Inference.ALL_TOKENS,
+            protocol=Protocols.HTTPS,
+            model_name=ModelFormat.CAIKIT,
+            use_default_query=True,
+            token=http_raw_inference_token,
+        )
+
+    @pytest.mark.skip(reason="Skipping test_cross_model_authentication_raw: RHOAIRFE-534")
```

Comment:
because auth fails (but a wrong code) i think that we should not skip the test but run the inference and check here that we fail (i.e do not use `verify_inference_response` )

Reply:
(i.e do not use verify_inference_response )
then what should be used ?

Reply:
if bug is open, use `run_inference_flow` and check we get is an expected error and not something else (i.e another bug)
else call `verify_inference_response`

Reply:
we have multiple test which test `expected error` is it really necessary?